src/GovUk.Education.ExploreEducationStatistics.Publisher/Functions/PublishTaxonomyFunction.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Publisher.Model;
+using GovUk.Education.ExploreEducationStatistics.Publisher.Models;
+using GovUk.Education.ExploreEducationStatistics.Publisher.Services.Interfaces;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using static GovUk.Education.ExploreEducationStatistics.Publisher.Model.PublisherQueues;
+
+namespace GovUk.Education.ExploreEducationStatistics.Publisher.Functions
+{
+    // ReSharper disable once UnusedType.Global
+    public class PublishTaxonomyFunction
+    {
+        private readonly IContentService _contentService;
+
+        public PublishTaxonomyFunction(IContentService contentService)
+        {
+            _contentService = contentService;
+        }
+
+        [FunctionName("PublishTaxonomy")]
+        // ReSharper disable once UnusedMember.Global
+        public async Task PublishTaxonomy(
+            [QueueTrigger(PublishMethodologyQueue)]
```

Comment:
This queue name isn't right

Reply:
Woops! Strangely I tested it and it seemed to work using this queue... Changed it anyway.

Reply:
🙂 Could this explain any of the intermittent test failures?

Reply:
Nope don't think so. The issues described above are very much to do with things being run in parallel.